### PR TITLE
fix click event on IE

### LINKED ISSUE #214: Click event not working on IE11
Versions:
- Photo-Sphere-Viewer : 3.4.0
- Three : 95.0
-  Internet Explorer 11

Click event is not working on IE. You can test it with the "equirectangular.html" example : no marker is added.

I think the bug is in the method "PSVUtils.getClosest" where "el.parentElement" is used. It turns out that on IE, SVGElement does not have a 'parentElement' property but a 'parentNode' one.

Thanks for your help

### PR DESCRIPTION
**Pull request by Biboba**
**Merge request checklist**

- [X] I read the [guidelines for contributing](https://github.com/mistic100/Photo-Sphere-Viewer/blob/master/.github/CONTRIBUTING.md)
- [X] I created my branch from `dev` and I am issuing the PR to `dev`
- [X] Unit tests are OK

This fixes #214. Tested on IE, Chrome, Edge, Safari, Firefox

### CODE REVIEW

File: src/js/PSVUtils.js [hunk 1]
```diff
@@ -263,7 +263,7 @@ PSVUtils.getClosest = function(el, selector) {
     if (matches.bind(el)(selector)) {
       return el;
     }
-  } while (!!(el = el.parentElement));
+  } while (!!(el instanceof SVGElement ? el = el.parentNode : el = el.parentElement));
```
Comment (mistic100) [inline src/js/PSVUtils.js]:
```js
el = el.parentElement || el.parentNode
```
would be a better solution in case there are other edge cases

**Comment by Biboba** [comment]
Did you test your solution ? I tried with that at the beginning and it will fail once at the body node

**Comment by mistic100** [comment]
@Biboba you are right, that's why I removed my comment :-)

**Comment by Biboba** [comment]
Great thanks ! If you could make a small release for that bug it would be perfect :)
Many thanks for this well documented repo.
